Makefile

```diff
@@ -36,6 +38,10 @@ generate: $(DEEPCOPY_TARGET) manifests
 manifests: $(CONTROLLER_GEN) $(TYPES_V1ALPHA1_TARGET)
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=operator paths="./pkg/..." output:crd:artifacts:config=config/crd/bases
 
+helm: $(CONTROLLER_GEN)
+	kustomize build config/helm | sed -e '/creationTimestamp/d' > helm/crds/bundle.yaml
+	tar zcvf kubernetes-event-exporter.tgz helm
```

Comment:
The tar package may be renamed `kube-events.tgz` as it includes not only the events exporter but also the events ruler.